packages/core/src/app/payment/paymentMethod/HostedCreditCardFieldsetValues.tsx

```diff
@@ -0,0 +1,34 @@
+import { CardInstrumentFieldsetValues } from "../storedInstrument";
```

Comment:
Just out of curiosity does this need to be a `.tsx` file?

Reply:
No reason. I just updated the file extension to be `.ts`